src/components/views/Credentials/Credential.tsx

```diff
@@ -3,7 +3,7 @@ import { Info, Person, VerifiedUser, Warning } from "@rimble/icons";
 import { Box, Flex, Pill, Table, Text } from "rimble-ui";
 import { VC } from "vc-schema-tools";
 import { CredentialBorder, CredentialContainer, Separator } from "./CredentialComponents";
-import { CredentialShare } from "./CredentialShare";
+import { CredentialShareButton } from "./CredentialShare";
```

Comment:
There were new designs for this section which I added. I think all updates on this are covered in my PR